azext_iot/common/utility.py

```diff
@@ -515,3 +516,11 @@ def compute_device_key(primary_key, registration_id):
         ).digest()
     )
     return device_key
+
+
+def generateKey(byteLength=32):
```

Comment:
Can we call this `generate_key` with the byteLength param renamed to `byte_length` in the spirit of pythonic naming?

Reply:
Fixed